templates/download/server/arm.html

```diff
@@ -5,124 +5,147 @@
-        image(
-        url="https://assets.ubuntu.com/v1/5eebb138-Servers.svg",
-        alt="",
-        width="294",
-        height="132",
-        hi_def=True,
-        loading="lazy",
-        ) | safe
-      }}
+    <div class="col">
+      <p>Ubuntu Server for&nbsp;ARM includes everything you are looking for in a server operating system, including:</p>
+      <hr class="p-rule" />
+      <ul class="p-list--divided u-no-padding--bottom u-no-margin--bottom">
+        <li class="p-list__item is-ticked">The LXD container hypervisor, giving you instant access to isolated, secured environments running with bare metal performance.</li>
+        <li class="p-list__item is-ticked">Application container technology based on Docker and Kubernetes, including FAN-based networking.</li>
+        <li class="p-list__item is-ticked">SQL and NoSQL databases including CouchDB, MySQL, PostgreSQL, Redis and SQLite.</li>
+        <li class="p-list__item is-ticked">A wide collection of web technologies, from HTTP servers to frameworks including Apache, Django, memcached, Nginx, WordPress and Varnish.</li>
+      </ul>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light is-deep">
-  <div class="row u-equal-height">
-    <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-        alt="",
-        width="281",
-        height="200",
-        hi_def=True,
-        loading="lazy",
-        ) | safe
-      }}
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Commercially supported and ready to deploy today</h2>
     </div>
-    <div class="col-8">
-      <h3>Commercially supported and ready to deploy today</h3>
-      <p>Pair your ARM server deployment with enterprise-grade 24x7 support with <a href="/pro">Ubuntu Pro</a> to get the SLA-backed  assurance that you are fully covered by our system and architecture experts &mdash; no matter what comes up.</p>
+    <div class="col">
+      <p>Pair your ARM server deployment with enterprise-grade 24/7 support with <a href="/pro">Ubuntu Pro</a> to get the SLA-backed assurance that you are fully covered by our system and architecture experts &mdash; no matter what comes up.</p>
       <p>Built for cutting-edge hardware, from the HP Moonshot range to standard form-factor certified systems, Ubuntu and ARM Server provide truly compelling economics for cloud-scale deployment.</p>
+      <hr class="p-rule" />
       <p><a href="/server/contact-us?product=server-arm">Contact us about your ARM server plans&nbsp;&rsaquo;</a></p>
     </div>
   </div>
 </section>
 
-{% with first_item="_download_server_arm_guide", second_item="_download_enterprise_support", third_item="_download_helping_hands" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+<section class="p-section--deep">
```

Comment:
We should think about making this section into a partial as it has appeared in a few places already

Reply:
The layout totally changed, that's why I removed this current partial, but yes I plan to do it once I finish the next few pages, so I'm sure that they all look the same and have the same content, as sometimes UX may make some changes that changes things in a way I may need to refactor the partial.